### ISSUE TITLE
Percentage based templates for split transactions

### ISSUE THREAD

**Issue by mtotschnig** [mtotschnig/MyExpenses#543]
User needs to create split transactions where parts are based on a percentage calculation and total amount differs. For example, income where a certain percentage is deducted as tax or fee.

**Comment by Farzat07** [comment]
Example receipt at a supermarket:

- Food items (8% VAT):

  - Oranges for ¥198

  - Bananas for ¥100

  - Total before tax is ¥298

  - Tax for food products is ¥23

- Non-food products (10% VAT):

  - Pencils for ¥100

  - Geometry tools for ¥248

  - Total before tax is ¥348

  - Tax for non-food products is ¥34

- Total paid is ¥(298 + 23 + 348 + 34) = ¥703

You can notice that the tax **totals** are usually rounded down.

The app should be able to split it as ¥213.28 for oranges, ¥107.72 for bananas, ¥109.77 for pencils, and ¥272.23 for geometry tools, either automatically or at least let us calculate it ourselves manually.

Currently though, the precision of splits is limited to that of the global settings - ie whole yens, which makes sense in normal transactions but not splits.

It would be great if both automatic and manual approaches are implemented, but as the automatic approach can be complicated and not applicable to all countries, it is completely understandable if it is not implemented. However, the manual approach (just optionally allowing more decimal places in splits only) should be implemented asap.

**Comment by mtotschnig** [comment]
@Farzat07 Do you know that you can adjust the number of fraction digits for each currency in the settings?

**Comment by Farzat07** [comment]
Yeah I know but what I wanted was to allow more decimal places for parts only. But nvm now I managed without it.

I'll reallocate my vote points to the deep category hierarchy feature.
